src/main/java/org/sandboxpowered/sandbox/fabric/util/wrapper/RenderUtilImpl.java

```diff
@@ -28,9 +28,9 @@ public void drawRepeating(int x, int y, int u, int v, int width, int height, flo
         GlStateManager.disableLighting();
         GlStateManager.disableFog();
         Tessellator tessellator_1 = Tessellator.getInstance();
-        BufferBuilder bufferBuilder_1 = tessellator_1.getBufferBuilder();
+        BufferBuilder bufferBuilder_1 = tessellator_1.getBuffer();
         GlStateManager.color4f(1.0F, 1.0F, 1.0F, 1.0F);
-        bufferBuilder_1.begin(7, VertexFormats.POSITION_UV_COLOR);
+        bufferBuilder_1.begin(7, VertexFormats.POSITION_COLOR_TEXTURE);
```

Comment:
This looks like it's gonna swap the argument orders of color and texture. If there's a `POSITION_TEXTURE_COLOR` I think that's what we want for the vertex setups below.